### PR TITLE
MINOR: Fix topology builder debug log message

### PR DESCRIPTION
**Pull request by tedyu**
This PR is follow up to the comment from #7969:

https://github.com/apache/kafka/pull/7969#discussion_r370902327

### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java [hunk 1]
```diff
@@ -1871,12 +1871,12 @@ boolean hasSubscriptionUpdates() {
 
     synchronized void updateSubscribedTopics(final Set<String> topics,
                                              final String logPrefix) {
-        log.debug("{}found {} topics possibly matching subscription", logPrefix, topics);
+        log.debug("{}found {} topics possibly matching subscription", logPrefix, topics.size());
```
Comment (ableegoldman) [inline streams/src/main/java/org/apache/kafka/streams/processor/internals/InternalTopologyBuilder.java:1874]: Thanks for the PR! I just noticed these two log messages are completely redundant; can we remove one of them, or combine into a single log?

**Comment by tedyu** [comment]
@ableegoldman @mjsax 
Please take another look.

**Review by ableegoldman — APPROVED** [review]
LGTM

**Pull request merged**